sirepo/comsol_register.py

```diff
@@ -6,51 +6,38 @@
 
 """
 from __future__ import absolute_import, division, print_function
-
 from pykern import pkconfig
 from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
 from sirepo import api_perm
 from sirepo import http_reply
 from sirepo import http_request
+from sirepo import smtp
 
 
-_mail = None
-
 @api_perm.allow_visitor
 def api_comsol():
     return http_reply.gen_redirect_for_anchor('/old#/comsol')
 
 
 @api_perm.allow_visitor
 def api_comsolRegister():
-    import flask
-    import flask_mail
     import sirepo.util
 
-    global _mail
-    if not _mail:
-        a = sirepo.util.flask_app()
-        a.config.update(
-            MAIL_USE_TLS=True,
-            MAIL_PORT=587,
-            MAIL_SERVER=cfg.mail_server,
-            MAIL_USERNAME=cfg.mail_username,
-            MAIL_PASSWORD=cfg.mail_password,
-        )
-        _mail = flask_mail.Mail(a)
-    req = http_request.parse_json()
-    msg = flask_mail.Message(
+    smtp.SMTP(
+        smtp_server=cfg.mail_server,
```

Comment:
This should be moved into smtp. we don't need two smtp logins and moreover, we probably only want to send from one address/name (support) so add those with appropriate defaults to smtp cfg. recipient config needs to stay in comsol. 

I don't think we need to deprecate the config in auth.email. Just remove it. I can add it to the rsconf db in advance of pushing. Nobody else is using smtp (for comsol or auth).

Reply:
https://github.com/radiasoft/sirepo/pull/2677/commits/5562b8e45c4790e870646d055a019adb1ceb5dc1